src/test/java/htsjdk/samtools/reference/IndexedFastaSequenceFileTest.java

```diff
@@ -41,6 +43,7 @@
 public class IndexedFastaSequenceFileTest extends HtsjdkTest {
     private static File TEST_DATA_DIR = new File("src/test/resources/htsjdk/samtools/reference");
```

Comment:
can you make these statics final while you're here?

Reply:
Done